Update gsctl internal api doc (#349)
giantswarm/docs: src/content/reference/gsctl/create-kubeconfig.md

overwrite this file will be required. The confirmation can be suppressed using
`--force`.

> The next example shows creation of a self-contained configuratio file with

Reviewer: ```suggestion
The next example shows the creation of a self-contained configuration file with
```

> internal Kubernetes API endpoint.

Reviewer 2: ```suggestion
an internal Kubernetes API endpoint.
```

```nohighlight
$ gsctl create kubeconfig -c w6wn8 --self-contained kubeconfig.yaml \
  --tenant-internal=true
```

> Here , the file `kubeconfig.yaml` will be created and will container internal

Reviewer: ```suggestion
Here, the file `kubeconfig.yaml` will be created and it will contain an internal
```

> Kubernetes API fqdn reference in `server` field. Currently, only AWS supports

Reviewer: Not sure everyone knows the acronym
```suggestion
Kubernetes API endpoint (FQDN) reference in the `server` field. Currently, only AWS supports
```

  config file. Credentials will be included. The file will contain only one
  user, one cluster, and one context. When this option is used, the default
  kubectl config file is not altered.

> - `--tenant-internal`: This option sets whether internal endpoint should be used

Reviewer 2: ```suggestion
- `--tenant-internal`: This option sets whether an internal endpoint should be used
```
